heralding/capabilities/handlerbase.py

```diff
@@ -88,7 +88,4 @@ def handle_session(self, gsocket, address):
                 logger.debug('Session timed out. ({0})'.format(session.id))
             finally:
                 self.close_session(session)
-                try:
```

Comment:
I remember we introduced this for a reason... But sadly I cannot remember which reason - let's try and see if we can do without it.